doc/source/train/examples.yml

```diff
@@ -1,22 +1,22 @@
-text: Below are examples for using Ray Train with a variety of frameworks and use cases.
+text: Below are examples for using Ray Train with a variety of frameworks and use cases. Ray Train makes it easy to scale out each of these examples to a large cluster of GPUs!
 columns_to_show:
   - frameworks
 examples:
-  - title: Train a Fashion MNIST Image Classifier with PyTorch
+  - title: Train an image classifier with PyTorch
     skill_level: beginner
     frameworks:
       - pytorch
     use_cases:
       - computer vision
     link: examples/pytorch/torch_fashion_mnist_example
-  - title: Train an MNIST Image Classifier with Lightning
+  - title: Train an image classifier with Lightning
     skill_level: beginner
     frameworks:
       - lightning
     use_cases:
       - computer vision
     link: examples/lightning/lightning_mnist_example
-  - title: Distributed Data Parallel Training with Hugging Face Accelerate
+  - title: Train a text classifier with HuggingFace Accelerate
```

Comment:
We've been using `Hugging Face` across the docs because that's how they style it. Though if we do want to merge it for readability we should do it across the docs.

Reply:
What Matt said. ^